src/AppBundle/DataFixtures/ORM/LoadFixtures.php

```diff
@@ -91,11 +96,114 @@ private function loadPosts(ObjectManager $manager)
             }
 
             $manager->persist($post);
+
+            $this->addReference('post-' . $i, $post);
```

Comment:
Good thinking, for while I am wondering why aren't we using multiple fixture classes (fixture class for each entity class) in this example?